services/ui-src/src/tests/Header.test.js

```diff
@@ -2,6 +2,16 @@ import React from "react";
 import { render } from "@testing-library/react";
 import Header from "../components/Header";
 
+window.matchMedia =
+  window.matchMedia ||
+  function () {
+    return {
+      matches: false,
+      addEventListener: function () {},
+      removeEventListener: function () {},
+    };
+  };
```

Comment:
its unfortunate... but we do need this here if we want to capture a snapshot of the header comp. the problem originates from a misconfiguration in the UsaBanner module in the cms design system package. I will log an issue with them